go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/agrea/ptr v0.0.0-20180711073057-77a518d99b7b
 	github.com/alicebob/miniredis/v2 v2.16.0
 	github.com/dapr/components-contrib v1.5.1-rc.1
+	github.com/dapr/dapr v1.5.1
```

Comment:
Hi, we don't really need to import 'dapr/dapr',which might cause dependency conflicts